raiden/blockchain_events_handler.py

```diff
@@ -172,10 +195,18 @@ def handle_channel_closed(raiden, event, current_block_number):
     )
 
     if channel_state:
+        # The from address is included in the ChannelClosed event as the
+        # closing_participant field
```

Comment:
Why is there commented out code here?

Reply:
I left it in there, together with the comment, explaining that unlike the previous handlers, this one doesn't need it.

Do you want me to remove it?

Reply:
yes please, leave the comment but you can remove the commented out code.

Reply:
done

Reply:
Hmm ... why? The last commit only removed comments. Didn't https://github.com/raiden-network/raiden/pull/1984/commits/6129dbade575b4d72141afc02905df1929d647c9 already run integration tests?

Reply:
Yes, but there were a couple of failures. Not sure of they were flaky.